src/tests/unit/test_tree_node_helpers.py

```diff
@@ -980,69 +1063,19 @@ def test_to_dict(self):
         self.assertEqual(item_1_actual, tree_node_to_dict(item_1))
 
     def test_is_storage_contained(self):
-        document_service = get_mock_document_service()
-
-        # Engine is a blueprint in a package called EnginePackage.
-        # We need to create 3 nodes: engine package, content list in engine package and the engine.
-        engine_entity = {
-            "name": "Engine",
-            "type": "dmss://system/SIMOS/Blueprint",
-            "extends": ["dmss://system/SIMOS/NamedEntity"],
-            "attributes": [
-                {"name": "hp", "type": "dmss://system/SIMOS/BlueprintAttribute", "attributeType": "string"}
-            ],
-        }
-        engine_blueprint_attribute = BlueprintAttribute(
-            name="content", attribute_type="dmss://system/SIMOS/Package", type="dmss://system/SIMOS/BlueprintAttribute"
-        )
-        engine_node: Node = Node(
-            key="0",
-            entity=engine_entity,
-            attribute=engine_blueprint_attribute,
-            blueprint_provider=document_service.get_blueprint,
-            recipe_provider=mock_storage_recipe_provider,
-        )
-
-        engine_package_content_bp_attribute = BlueprintAttribute(
-            name="content", attribute_type="object", type="dmss://system/SIMOS/BlueprintAttribute"
-        )
-        engine_package_content: ListNode = ListNode(
-            key="content",
-            attribute=engine_package_content_bp_attribute,
-            entity=[engine_entity],
-            blueprint_provider=document_service.get_blueprint,
-            recipe_provider=mock_storage_recipe_provider,
-        )
-
-        engine_package_entity = {
-            "_id": "26d94353-3ff0-4b7d-bf06-86f006dd6f7b",
-            "type": "dmss://system/SIMOS/Package",
-            "name": "EnginePackage",
-            "isRoot": False,
-            "content": [engine_entity],
+        engine_package_node = get_engine_package_node()
+        engine_ref_node = engine_package_node.children[0].children[0]
+
+        assert engine_ref_node.storage_contained is True
```

Comment:
before, this test used to check storage_contained is True.

but I think that is correct anymore after we added the new reference system (a package's content list contains references)

Reply:
Agreed, the test is wrong. Content of a package should not be storage contained, only model contained.